src/test/java/uk/gov/companieshouse/itemhandler/kafka/OrdersKafkaConsumerTest.java

```diff
@@ -25,96 +31,145 @@
 public class OrdersKafkaConsumerTest {
```

Comment:
Do these tests examine how many retries can take place?

Reply:
Since this story does not implement email sending (processing) using a `@Retryable` method. It does not test retries. Also, because of AOP requirements as described in the PR's description, the `@Retryable` implementation will need to be addressed using a separate class.

Reply:
As part of resilience logic, however, this story can implement an interface that defines a `@Retryable` method (to be implemented by the email sending/message processing logic).

Reply:
I just made a point somewhere above about documenting the way the Spring Retry `@Retryable` annotation is expected to be used here. I think one or more integration tests demonstrating how this works and that it works as expected would help a lot with this.

Reply:
This has now changed as the resilience logic no longer relies on @Retryable.